content/05.discussion.md

```diff
@@ -1,8 +1,45 @@
-# Discussion
-Here mention why performnace increases in the beginning for the generative model then decreases
+## Discussion
 
-Discuss discriminator model performance given generative model
-
-Mention Take home messages
-
-1. have a centralized set of negative label functions and focus more on contstructing positive label functions
+We tested the feasibility of re-using label functions to extract relationships from literature.
+Through our sampling experiment we found that adding relevant label functions increases prediction performance.
+This can been seen in the on-diagonals of generative model performance plots (Figure {@fig:generative_model_auroc} and {@fig:generative_model_auprc}).
+An interesting finding is performance increases when text patterns and domain heuristic label functions are added on top of the distant supervision label functions. 
+This increase happens because curated databases have low coverage and end up missing information.  
+Some relationships are easier to predict than others.
+For example, we achieved a significantly high area under the receiver operating curve (AUROC) and area under the precision recall curve (AUPR) for Compound-treats-Disease (CtD) (Figure {@fig:generative_model_auroc} and {@fig:generative_model_auprc}).
+Reason for high performance is that wording is pretty straight forward when abstracts talk about this particular edge type.
+Regarding Disease-associates-Gene (DaG), this edge type has a broad range of keywords that can infer associations.
```

Comment:
There are a lot of assertions here and I didn't see evidence for them as I read the paper. For these claims where does the evidence come from? If it's not in the paper, it either needs to be in the paper or the claim should be removed.